avm/res/network/load-balancer/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -1,5 +1,8 @@
 targetScope = 'subscription'
 
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Well-Architected Framework.'
```

Comment:
Note, adding the metadata requires a rerun of the Readme generation. I'd advice to do that and then run the pipeline with only the static tests enabled to check if all passes